src/families/filecoin/specs.ts

```diff
@@ -16,7 +16,7 @@ const filecoinSpecs: AppSpec<Transaction> = {
     appName: "Filecoin",
   },
 
-  testTimeout: 2 * 60 * 1000,
+  testTimeout: 5 * 60 * 1000,
```

Comment:
This seems like a lot, users would need to wait 5min to see their transactions confirmed?
Any chance confirmed transactions get seen sooner in the future?

Reply:
In practice, it took less than 3 minutes. A little more than 2 minutes. I think in the future we will improve it tuning configuration parameters to make it faster.